fineract-provider/build.gradle

```diff
@@ -276,7 +278,7 @@ configurations {
 }
 
 tasks.withType(JavaCompile) {
-    options.compilerArgs += ["-Xlint:unchecked","-Xlint:cast","-Werror"] // TODO FINERACT-959 (gradually) enable -Xlint:all (see "javac -help -X")
+    options.compilerArgs += ["-Xlint:unchecked","-Xlint:cast"] // TODO FINERACT-959 (gradually) enable -Xlint:all (see "javac -help -X")
```

Comment:
wait waaaaat... why are we regressing on FINERACT-959 to progress with FINERACT-822? That doesn't seem right, to me - and worth a discussion...  :smiling_imp: I suspect that you may be doing this because Error Prone gives warnings? IMHO we should either fix them all, as part of this - or change EP's config to shut up about particular ones which are obviously wrong (similar to what I have done in another project, have you seen the link in FINERACT-822)...  How many are there?

Reply:
@vorburger thanks for the link I will configure error prone to help with the 1000 warnings.